code/customization/models/libSupport/keras.q

```diff
@@ -9,9 +9,12 @@
 // @param mname {sym} name of the model being applied
 // @return      {int;float;bool} the predicted values for a given model as applied to input data
 models.keras.fitScore:{[data;seed;mname]
-  mdl:get[".automl.models.keras.",string[mname]".model"][data;seed];
-  mdl:get[".automl.models.keras.",string[mname],".fit"][data;mdl];
-  get[".automl.models.",string[mname],".predict"][data;mdl]
+  if[mname~`multi;
+    data[;1]:models.i.npArray@'flip@'value@'.ml.i.onehot1 each data[;1]];
+  dataDict:`xtrain`ytrain`xtest`ytest!raze data;
+  mdl:utils.qpyFuncSearch[".automl.models.keras.",string[mname],".model"][dataDict;seed];
```

Comment:
These can be get only, Python functions can't be namespaced within a `dot` namespace, so they won't be callable.